pkg/providers/instancetype/types.go

```diff
@@ -240,7 +241,7 @@ func ephemeralStorage(amiFamily amifamily.AMIFamily, blockDeviceMappings []*v1be
 func awsPodENI(ctx context.Context, name string) *resource.Quantity {
 	// https://docs.aws.amazon.com/eks/latest/userguide/security-groups-for-pods.html#supported-instance-types
 	limits, ok := Limits[name]
-	if awssettings.FromContext(ctx).EnablePodENI && ok && limits.IsTrunkingCompatible {
+	if settings.FromContext(ctx).EnablePodENI && ok && limits.IsTrunkingCompatible {
```

Comment:
Is this always guaranteed to be set even if the settings ConfigMap doesn't exist>?

Reply:
Nope, discussed offline that this should be modified to return the default value in the case that it doesn't exist.